### PR TITLE
Update config error message for `outDir` infinity loop error

### PR DESCRIPTION
**Pull request by andremralves**
## Changes

Related to: https://github.com/withastro/astro/pull/8152

Just updating the error message with the suggested one. 

## Testing

n/a

## Docs
n/a

### CODE REVIEW

**Comment by ematipico** [comment]
cc @withastro/maintainers-docs 

Could you please review the error message?

File: packages/astro/src/core/config/schema.ts
Comment (ElianCodes) [inline packages/astro/src/core/config/schema.ts:413]:
```suggestion
				'The value of `outDir` must not point to a path within the folder set as `publicDir`, this will cause an infinite loop',
```

File: packages/astro/test/units/config/config-validate.test.js
Comment (ElianCodes) [inline packages/astro/test/units/config/config-validate.test.js:77]:
```suggestion
			'The value of `outDir` must not point to a path within the folder set as `publicDir`, this will cause an infinite loop'
```

**Review by ElianCodes — COMMENTED** [review]
Just a small nit! Good to merge after suggestion merge!

**Review by ematipico — APPROVED** [review]

**Pull request merged**
